pageserver/src/tenant/timeline/eviction_task.rs

```diff
@@ -120,6 +120,13 @@ impl Timeline {
                 }
                 let elapsed = start.elapsed();
                 crate::tenant::tasks::warn_when_period_overrun(elapsed, p.period, "eviction");
+                crate::metrics::EVICTION_ITERATION_DURATION
+                    .get_metric_with_label_values(&[
+                        &format!("{}", p.period.as_secs()),
+                        &format!("{}", p.threshold.as_secs()),
```

Comment:
Unfortunate to-stringing... But perhaps does not matter enough to hoist this to upper level and do the tricks which already panicked in staging again.

Reply:
Yeah. I didn't want to go through the trouble of building the equivalent of `EvictionsWithLowResidenceDurationBuilder` for this metric.